features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+namespace RoaveTest\BehatPsrContainer;
+
+use Behat\Behat\Context\Context;
+
+final class FeatureContext implements Context
+{
+    /**
+     * @var TestService
+     */
+    private $testService;
+
+    public function __construct(TestService $testService)
+    {
+        $this->testService = $testService;
+    }
+
+    /**
+     * @Given /^I have a Zend\\ServiceManager container$/
+     */
+    public function iHaveAZendServiceManagerContainer()
+    {
+    }
+
+    /**
+     * @When /^I instantiate a context$/
+     */
+    public function iInstantiateAContext()
+    {
```

Comment:
Yep, but it's pretty tricky to test Behat extensions - you normally end up resorting to something where you're checking from inside behat that the extension's doing its thing.

A more complete strategy would be to run another Behat process end-to-end, but that's also a bit clunky.

This is better than no tests at least